pkg/util/github/github_test.go

```diff
@@ -90,79 +84,57 @@ func TestDownloadOc(t *testing.T) {
 	defer minitesting.ResetDefaultRoundTripper()
 
 	testDir, err := ioutil.TempDir("", "minishift-test-")
-	if err != nil {
-		t.Fatal(err)
-	}
+	assert.NoError(t, err)
 	defer os.RemoveAll(testDir)
 
 	for _, testAsset := range assetSet {
 		err = DownloadOpenShiftReleaseBinary(testAsset.binary, testAsset.os, testAsset.version, testDir)
-		if err != nil {
-			t.Fatal(err)
-		}
+		assert.NoError(t, err)
 
 		expectedBinaryPath := filepath.Join(testDir, testAsset.binary.String())
 		if testAsset.os == minishiftos.WINDOWS {
 			expectedBinaryPath += ".exe"
 		}
 		fileInfo, err := os.Lstat(expectedBinaryPath)
-		if err != nil {
-			t.Fatal(err)
-		}
+		assert.NoError(t, err)
 
 		if runtime.GOOS != "windows" {
 			expectedFilePermissions := "-rwxrwxrwx"
-			if fileInfo.Mode().String() != expectedFilePermissions {
-				t.Fatalf("Wrong file permisisons. Expected %s. Got %s", expectedFilePermissions, fileInfo.Mode().String())
-			}
+			assert.Equal(t, expectedFilePermissions, fileInfo.Mode().String())
 		}
 
 		err = os.Remove(expectedBinaryPath)
-		if err != nil {
-			t.Fatalf("Unable to delete %s", expectedBinaryPath)
-		}
+		assert.NoError(t, err)
 	}
 }
 
 func TestInvalidVersion(t *testing.T) {
 	EnsureGitHubApiAccessTokenSet(t)
 
 	testDir, err := ioutil.TempDir("", "minishift-test-")
-	if err != nil {
-		t.Fatal(err)
-	}
+	assert.NoError(t, err)
 	defer os.RemoveAll(testDir)
 
 	dummyVersion := "foo"
 	err = DownloadOpenShiftReleaseBinary(OPENSHIFT, minishiftos.WINDOWS, dummyVersion, testDir)
-	if err == nil {
-		t.Fatal("There should have been an error")
-	}
+	assert.Error(t, err)
 
-	expectedErrorMessage := fmt.Sprintf("Cannot get the OpenShift release version %s", dummyVersion)
-	if !strings.HasPrefix(err.Error(), expectedErrorMessage) {
-		t.Fatalf("Expected error: '%s'. Got: '%s'\n", expectedErrorMessage, err.Error())
-	}
+	expectedErrorMessage := fmt.Sprintf("Cannot get the OpenShift release version %s: GET https://api.github.com/repos/openshift/origin/releases/tags/foo: 404 Not Found []", dummyVersion)
```

Comment:
Kind of feels like modification to make the test pass, but this was necessary to use the _EqualError_ assertion.

Reply:
@anjannath , its fine.

Reply:
Okay, @budhrg could you also do a review of the changes at your leisure. :)